src/App.js

```diff
@@ -1,14 +1,24 @@
 import "./App.css";
-import logo from "./logo.png";
+import { BrowserRouter, Routes, Route, } from "react-router-dom";
 import Header from "./components/Header";
-import Homepage from "./pages/home/Homepage";
+import HomePage from "./pages/HomePage/HomePage";
+import { ProductPage } from "./pages/ProductsPage/ProductPage";
+import { CartPage } from "./pages/CartPage/CartPage";
+import { LoginPage } from "./pages/LoginPage/LoginPage";
+import { WishListPAge } from "./pages/WishListPage/WishListPAge";
```

Comment:
Here, you can import in single line by making index.js in pages and exporting from there.This is called barrel importing.